apps/ledger-live-mobile/src/components/FirmwareUpdateBanner.tsx

```diff
@@ -68,12 +68,15 @@ const FirmwareUpdateBanner = () => {
       lastSeenDevice.deviceInfo,
       lastSeenDevice.modelId,
     );
+  const isDeviceConnectedViaUSB = lastConnectedDevice?.wired;
```

Comment:
In this case it would work either way, but I agree with you, I feel it's always better to keep the types of the variables sane and not rely too much on the "falsy" feature of JavaScript :p

I've updated it